little-books/src/App.tsx

```diff
@@ -0,0 +1,122 @@
+import { useEffect, useState, useRef } from "react";
+import BlogDetail from "./components/BlogDetail/BlogDetail";
+import BlogLists from "./components/BlogLists/BlogLists";
+import Sidebar from "./components/Sidebar/Sidebar";
+import { fetchData } from "./services/fetch.service";
+import "./styles/main.scss";
+import { useAppDispatch, useAppSelector } from "./hooks";
+import { fetchBlogs, getFilteredBlogs } from "./redux/blogs/blogSlice";
+import AddBlog from "./components/AddBlog/AddBlog";
+import { BlogType } from "./types/blog.type";
+import ViewMembers from "./components/ViewMembers/ViewMembers";
+import Modal from "./components/Modal/Modal";
+import Search from "./components/Seach/Search";
+import { APP_CONSTANTS } from "./constants/constants";
+import { Member } from "./types/members.type";
+import React from "react";
+
+function App() {
+	const [darkMode, setDarkMode] = useState(false);
+	const [members, setMembers] = useState<Member[]>([]);
+	const dispatch = useAppDispatch();
+	const responseData = useAppSelector((state) => state.blogs);
+
+	const currentBlog = useAppSelector((state) => state?.blogs?.currentBlog);
+	const [blog, setBlog] = useState(currentBlog);
+	const [modalStates, setModalStates] = useState({
+		addBlogModal: false,
+		viewMembersModal: false,
+	});
```

Comment:
Strict typecheck for states whenever possible

Reply:
resolved